connectors/class-connector-blogs.php

```diff
@@ -74,7 +74,7 @@ public function get_context_labels() {
 
 			foreach ( $blogs as $blog ) {
 				$blog_details   = get_blog_details( $blog->blog_id );
-				$key            = sanitize_key( $blog_details->blogname );
```

Comment:
Looks like context label keys have to be globally unique so how about we used something like `sprintf( 'blog-%d', $blog->blog_id );` for the key? That should be safe in every instance.

Would this break any of the existing records and the way they're filtered?

Reply:
You are right @kasparsd `blog_id` would be safer to use rather than `blog_name` !

Reply:
This didn't break existing records on my testing.